sklearn/gaussian_process/gpc.py

```diff
@@ -116,8 +116,11 @@ def optimizer(obj_func, initial_theta, bounds):
 
     Attributes
     ----------
-    X_train_ : array-like, shape = (n_samples, n_features)
-        Feature values in training data (also required for prediction)
+    X_train_ : posssible values:
```

Comment:
This won't render correctly in html. Perhaps just say "sequence of length n_samples" and expand in the description

Reply:
Fixed accordingly.